### PR TITLE
Add XDebug section to Readme.md

### PR DESCRIPTION
**Pull request by Kanti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,24 @@ if ($_SERVER['TYPO3_CONTEXT'] === 'Development/docker') {
 }
 ```
 
+## Xdebug
+
+if you like to use xdebug, you can enable it via the `.env` file:
+````env
```
Comment (Cyb10101) [inline README.md]:
```suggestion

```env
```

File: README.md [hunk 2]
```diff
@@ -110,6 +110,24 @@ if ($_SERVER['TYPO3_CONTEXT'] === 'Development/docker') {
...
+PHP_DEBUGGER=xdebug
+XDEBUG_REMOTE_HOST=10.50.1.200
+````
```
Comment (Cyb10101) [inline README.md]: Change ```` to ```
Add new line after.

File: README.md [hunk 3]
```diff
@@ -110,6 +110,24 @@ if ($_SERVER['TYPO3_CONTEXT'] === 'Development/docker') {
...
+Where `XDEBUG_REMOTE_HOST` is the ip of your IDE.
+
+**CLI:** run any php script with this prefixed:
+````
+XDEBUG_CONFIG="remote_enable=1" php #...
```
Comment (Cyb10101) [inline README.md]:
```suggestion
XDEBUG_CONFIG="remote_enable=1" php script.php
```

File: README.md [hunk 4]
```diff
@@ -110,6 +110,24 @@ if ($_SERVER['TYPO3_CONTEXT'] === 'Development/docker') {
...
+
+if you like to use xdebug, you can enable it via the `.env` file:
+````env
+PHP_DEBUGGER=xdebug
+XDEBUG_REMOTE_HOST=10.50.1.200
+````
+Where `XDEBUG_REMOTE_HOST` is the ip of your IDE.
+
+**CLI:** run any php script with this prefixed:
+````
```
Comment (Cyb10101) [inline README.md]: Add new line before.
Change ```` to ```

File: README.md [hunk 5]
```diff
@@ -110,6 +110,24 @@ if ($_SERVER['TYPO3_CONTEXT'] === 'Development/docker') {
...
+XDEBUG_CONFIG="remote_enable=1" php #...
+````
```
Comment (Cyb10101) [inline README.md]: Change ```` to ```

**Comment by Cyb10101** [comment]
Maybe useful for this pull request: Configure xDebug in PhpStorm
https://github.com/pluswerk/docker-vm/blob/master/usage.md#configure-xdebug-in-phpstorm

**Comment by Kanti** [comment]
closed in favor of #46

**Pull request closed without merging**